website/docs/prysm-usage/web-interface.md

```diff
@@ -88,11 +88,7 @@ where you replace `user@host_ip` with the user and host ip address of the remote
 
 You can visualize your beacon node and validator client logs from the web interface easily by navigating to `System Process -> Logs` on the left-hand sidebar.
 
-<<<<<<< HEAD
-We recommend going through the ["imported wallet"](/docs/wallet/nondeterministic) route, and importing your keys you obtained during the [eth2 launchpad deposit-cli](https://launchpad.ethereum.org/) process, as this is the most secure setup. Upon completing wallet creation, you will be redirected to your main dashboard, where you can see several important items such as your recent validating performance or your beacon node's sync status.
-=======
 ![Logs](/img/logs.png "Logs")
```

Comment:
lets shift the logs above this paragraph

Reply:
Not 100% sure what you mean, assuming move them closer so got rid of the newline inbetween.

Reply:
Ah no I meant put the image below this chunk:
> You can visualize your beacon node and validator client logs from the web interface easily by navigating to `System Process -> Logs` on the left-hand sidebar.

Otherwise it is out of place

Reply:
Ah that makes a lot more sense. Thanks. Done.